benchmarks/tables/fh-table.jl

```diff
@@ -6,16 +6,16 @@ display_sol = true
 random_seed = 12345
 Random.seed!(random_seed)
 
-cstr = false
+cstr = true
```

Comment:
Why change a problem in this PR?

Reply:
I had better results when changing $\lambda$ and the norm of the regularizer, but I set `cstr` back to false in my last commit (I set it manually to true when I want to generate tables with constrained-FH).